src/status_im/ui/components/list_item/styles.cljs

```diff
@@ -1,24 +1,86 @@
 (ns status-im.ui.components.list-item.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
   (:require [status-im.ui.components.colors :as colors]))
 
-(defn container [small?]
-  {:height           (if small? 52 64)
+(defn container [type]
+  {:min-height         (cond
+                         (= type :small)          52
+                         (= type :section-header) 40
+                         :else                    64)
+   :flex               1
+   :flex-direction     :row
+   ;; :background-color   colors/red-transparent-10
+   :justify-content    :flex-start
+   :padding-horizontal 16
+   :padding-top        (cond
+                         (= type :default) 10
+                         :else             14)
+   :padding-bottom     (cond
+                         (= type :small)          14
+                         (= type :section-header) 4
+                         :else                    10)
+   :align-items        :center})
+
+(defnstyle icon-container [color]
+  {:border-radius    20
+   :width            40
+   :height           40
    :align-items      :center
-   :flex-direction   :row
-   :background-color :white})
+   :justify-content  :center
+   :background-color (or color colors/blue-light)})
+
+(defn icon [color]
+  {:color     (or color colors/blue)
+   :font-size 16})
 
-(defn title [small? subtitle]
-  (merge (when-not small?
-           {:typography :title})
-         (when subtitle
+(defn title-prefix [type icon? subtitle content]
+  (merge {:margin-left (if icon? 2 0)
+          ;; :background-color colors/red-transparent-10
+          :line-height 22
+          :align-self  :stretch}
+         (when (= type :default)
+           {:typography  :title
+            :height      20
+            :line-height 20})
+         (when (or subtitle content)
            {:typography :main-medium})))
 
-(def subtitle
-  {:margin-top  4
-   :color       colors/gray})
+(defn title [type icon? title-prefix subtitle content title-row-accessory]
+  (merge {:margin-left (if icon? 2 0)
+          :flex        1
+          :flex-grow   1
+          ;; :background-color colors/red-transparent-10
+          :flex-shrink 1
+          :line-height 22
+          :align-self  :stretch}
+         (when (= type :default)
+           {:typography  :title
+            :height      20
+            :line-height 20})
```

Comment:
are you sure you want to overwrite the line-height here it is calculated based on the height, for 20 it is 26

Reply:
I am pretty sure. Again tried to follow figma design spec to the tee, except when I used best judgement and did something slightly different. Even when spec was not followed to the tee, did follow the `intent` very closely. Even captured screen from app, exported design from figma, overlayed in Photoshop, checked for `pixel-perfectness` 😛 . Doing so, found out that difference in font rendering in different devices can make `pexel-perfectness` almost impossible. But I tried to get it close.

I am changing/consolidating the logic a bit more so the intent is clearer.